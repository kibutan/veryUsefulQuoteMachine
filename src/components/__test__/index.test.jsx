import { render, screen } from "@testing-library/react";
import Index from "../../pages/index";
import "@testing-library/jest-dom";

describe("Index", () => {
  test("renders", () => {
    render(<Index />);
    expect(screen.getByText("Quote")).toBeInTheDocument();
  });
});
