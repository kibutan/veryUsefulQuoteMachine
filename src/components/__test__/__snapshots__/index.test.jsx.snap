// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders 1`] = `
<div
  class="css-0"
>
  Quote
</div>
`;
